nestkernel/music_manager.cpp

```diff
@@ -106,9 +105,23 @@ MUSICManager::enter_runtime( double h_min_delay )
   {
     music_runtime = new MUSIC::Runtime( music_setup, h_min_delay * 1e-3 );
   }
-#endif
 }
 
+#else /* #ifdef HAVE_MUSIC */
+
+void
+MUSICManager::init_music( int*, char** )
```

Comment:
A bit unusual, but needs another *, see Travis log:
```suggestion
MUSICManager::init_music( int*, char*** )
```

Reply:
Ah, yes. the argument name in the `HAVE_MUSIC` variant is `argv[]`, which adds this additional level of arrayness.